daemon/cluster/executor/container/container.go

```diff
@@ -401,6 +401,7 @@ func (c *containerConfig) hostConfig(deps exec.VolumeGetter) *enginecontainer.Ho
 		Sysctls:        c.spec().Sysctls,
 		CapAdd:         c.spec().CapabilityAdd,
 		CapDrop:        c.spec().CapabilityDrop,
```

Comment:
Not for this PR, but looking at this list, I'm wondering if there's other properties that are actually wired up, but that were forgotten to be included in the swagger.

If that's the case, we should fix that 🤔